### ISSUE TITLE
Keyboard navigation in Device Settings skips the first link (Firefox)

### ISSUE THREAD

**Issue by sairina** [learningequality/kolibri#9188]
## Observed behavior
When in device settings, the focus passes over the first link.
![2022-03-14 14 52 09](https://user-images.githubusercontent.com/13563002/158267485-514a9840-8060-45cb-b55c-07c9419c849c.gif)

## Errors and logs
None

## Expected behavior
The link should be the first element focused on this page.

## User-facing consequences
Keyboard user is unable to click on link.

## Steps to reproduce
1. Sign in as admin
2. In Device, go to Settings tab
3. Use the keyboard to navigate. Notice that you cannot access the "You can also configure facility settings" link via keyboard

## Context
Kolibri 0.15
Firefox

**Comment by marcellamaki** [comment]
Possible Mac settings issue 

![firefox-no-skipped-link.gif](https://images.zenhubusercontent.com/6047f6e691103a78d3ad0ace/607c7c32-cdef-40c5-a583-a8e551a23b33)

**Comment by marcellamaki** [comment]
cc @radinamatic -- to double check before closing

**Comment by pcenov** [comment]
Yeah, this is also a Mac configuration issues - works fine for me on Safari, Firefox and Chrome on Mac after enabling the correct settings.
